### DISCUSSION TITLE
Cor_heatmap annotated

### DISCUSSION THREAD

**Discussion by mlkleinbest** [david-barnett/microViz#154] [General]
Hi,

I cannot figure out if it's possible to annotate the cor_heatmap similarly to the comp_heatmap annotations where samples are highlighted with a bar across the top of the heatmap i.e., from the annotated heatmap tutorial:

sample_anno = [sampleAnnotation](https://david-barnett.github.io/microViz/reference/sampleAnnotation.html)(
      State1 = [anno_sample](https://david-barnett.github.io/microViz/reference/anno_sample.html)("DiseaseState"),
      col = [list](https://rdrr.io/r/base/list.html)(State1 = cols), border = FALSE,
      State2 = [anno_sample_cat](https://david-barnett.github.io/microViz/reference/anno_sample_cat.html)("DiseaseState", col = cols)
    ),
    sample_seriation = "Identity" # suppress sample reordering

**Comment by david-barnett** [comment]
Hi, sample annotations would not make sense for cor_heatmap, as neither the rows or columns represent samples. Instead they are metadata variables and microbes.
Hope this helps clarify :)
